### DISCUSSION TITLE
Seeking Paired Image and Video Dataset for Image-to-Video Generation

### DISCUSSION THREAD

**Discussion by OldWhitee** [community/community#144169] [Programming Help]
### Select Topic Area

Question

### Body

Hello everyone! I’m currently working on an image-to-video generation project and am looking for a dataset with paired images and videos for model training and evaluation. Ideally, the dataset would include images and corresponding videos of the same scenes, or videos with keyframes that can be used as reference images for generating consistent video output.

I've tried searching for publicly available datasets but haven’t found one that meets these requirements. If anyone knows of a suitable dataset or can suggest potential data sources, I would greatly appreciate it! Any guidance, suggestions, or links would be incredibly helpful.

Thank you all for your help!

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @OldWhitee, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
